### DISCUSSION TITLE
It is recommended to upload small files in collapse

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#66] [Ideas]
It can be reported to upload a smaller collapse DMP file to establish an event number; report to the construction of testers to follow up and analyze.

**Comment by gz83** [comment]
> It can be reported to upload a smaller collapse DMP file to establish an event number; report to the construction of testers to follow up and analyze.

@whxuezhe 
If you encounter a crash, please describe in detail how the crash occurred and include the URL (if there is one).  In addition, please provide the crash log according to the guidelines on the following page for further verification.

https://www.chromium.org/developers/crash-reports/

If you don't encounter the crash problem, please don't open an issue or discussion, it will only take up resources and waste our time.

**Reply by whxuezhe** [reply]
There are DMPs, but they are not trying to reproduce in the construction again; it is known that Google Golden Silk has emerged steadily, and in the core of the Google browser in the upstream, it cannot be opened and landed on the domestic card rice forum. I am disturbed, occupying your precious time and resources.
This type of browser DMP with only less than 10m, do not want to store cloud disks, and stick URLs. I think it is not a good move!

**Reply by gz83** [reply]
> There are DMPs, but they are not trying to reproduce in the construction again; it is known that Google Golden Silk has emerged steadily, and in the core of the Google browser in the upstream, it cannot be opened and landed on the domestic card rice forum. I am disturbed, occupying your precious time and resources. This type of browser DMP with only less than 10m, do not want to store cloud disks, and stick URLs. I think it is not a good move!

@whxuezhe If vallina Chromium cannot open the forum or webpage you mentioned, please report the problem you encounter to Google in crbug with an accurate English description.

In addition, the crash log file is very important to solve the crash problem, it will contain some information that triggered the crash.
